### PR TITLE
Improved tutorial

### PR DESCRIPTION
**Pull request by ens-emily**

### CODE REVIEW

File: docs/htdocs/info/docs/api/variation/variation_tutorial.html [hunk 1]
```diff
@@ -919,18 +934,20 @@ <h2 id="variant_sets">Variant sets</h2>
...
+    <p>1000 Genomes Project phase 3 data is accessed via VCF files stored on Ensembl's FTP server. 
+    The API makes this access seamless, though it requires you install some extra software dependencies detailed on the 
+    <a href="/info/docs/api/api_installation.html#variation">API installation page</a>. 
+    See also this <a href="http://www.ensembl.info/blog/2015/06/18/1000-genomes-phase-3-frequencies-genotypes-and-ld-data/" rel="external">blog post</a>.
     </p>
   </div>
 </div>
 
-<h2 id="alleles">Alleles and frequencies</h2>
+<h3>Alleles and frequencies</h3>
```
Comment (sarahhunt) [inline docs/htdocs/info/docs/api/variation/variation_tutorial.html]: Most variants in the Ensembl Variation database have associated allele frequencies. => Many variants available through the Ensembl Variation API have allele frequency information.

File: docs/htdocs/info/docs/api/variation/variation_tutorial.html [hunk 2]
```diff
@@ -919,18 +934,20 @@ <h2 id="variant_sets">Variant sets</h2>
...
 
 
-<br /><hr />
 <h2 id="ag" style="font-size:large">Alleles and Genotypes</h2>
 
 <div style="width:800px" class="info">
   <h3>Requirement for 1000 Genomes data and other large genotyping projects</h3>
   <div class="message-pad">
-    <p>1000 Genomes Project phase 3 data is accessed via VCF files stored on Ensembl's FTP server. The API makes this access seamless, though it requires you install some extra software dependencies detailed on the <a href="/info/docs/api/api_installation.html#variation">API installation page</a>. See also this <a href="http://www.ensembl.info/blog/2015/06/18/1000-genomes-phase-3-frequencies-genotypes-and-ld-data/" rel="external">blog post</a>.
+    <p>1000 Genomes Project phase 3 data is accessed via VCF files stored on Ensembl's FTP server.
```
Comment (sarahhunt) [inline docs/htdocs/info/docs/api/variation/variation_tutorial.html]: '1000 Genomes Project phase 3 data is accessed via VCF files stored on Ensembl's FTP server.'
=>'Genotype and frequency data for a number of large scale projects, including the 1000 Genomes Project, can be accessed direct VCF files stored on Ensembl's FTP server'

File: docs/htdocs/info/docs/api/variation/variation_tutorial.html [hunk 3]
```diff
@@ -983,21 +1000,27 @@ <h2 id="alleles">Alleles and frequencies</h2>
   <h3>Retrieve external data (e.g. 1000 Genomes phase 3)</h3>
   <div class="message-pad">
     <p>To retrieve external data e.g. from the 1000 Genomes phase 3 in human, remember to set the flag <b>"use_vcf"</b> to <span style="color:#F00;font-weight:bold">1</span> before making a call to retrieve alleles, genotypes or LD data. See an example in the following code:</p>
-    <pre class="code sh_perl">my $variation_adaptor = $registry-&gt;get_adaptor("human", "variation", "variation");
+    <pre class="code sh_perl">
```
Comment (sarahhunt) [inline docs/htdocs/info/docs/api/variation/variation_tutorial.html]: This needs updating to reflect that it's not just 1KG with data outside the db. There are more results if you switch VCF usage on for this example too.

How about 'To retrieve external data e.g. from the 1000 Genomes phase 3 in human, remember to set the flag ' =>
Additional frequency data is available via our VCF API extensions. Add the line
$variation_adaptor->db->use_vcf(1)
in the aboue example to retrieve frequencies from the WSI  Mouse Genomes Project

File: docs/htdocs/info/docs/api/variation/variation_tutorial.html [hunk 4]
```diff
@@ -1264,11 +1321,10 @@ <h3>Requirement</h3>
 <li>the estimated r<sup>2</sup> value is lower than the cut-off threshold in the calculation software (the threshold is 0.05)</li>
 </ul>
 <p> Looking at other associated data will help to narrow down the reason. If there are variants in the region spanned by the LDFeatureContainer 
-with genotypes with a minor allele frequency greater than zero, then assume the r<sup>2</sup> is &lt; 0.05, which essentially means no correlation. </p>
+with genotypes with a minor allele frequency greater than zero, then assume the r<sup>2</sup> is < 0.05, which essentially means no correlation. </p>
 
 
-<br />
-<h2 id="strains">Specific strain information</h2>
+<h3 id="strains">Specific strain information</h3>
```
Comment (sarahhunt) [inline docs/htdocs/info/docs/api/variation/variation_tutorial.html]: I'd loose the 'With the apparition of the new technologies, one of the new functionalities that the'

File: docs/htdocs/info/docs/api/variation/variation_tutorial.html [hunk 5]
```diff
@@ -17,92 +18,17 @@ <h2 id="intro" style="font-size:large">Introduction</h2>
...
-my $variation = $variation_adaptor-&gt;fetch_by_name('rs1333049');
-</pre>
-
 <p>
-	The <b>get_adaptor</b> method will automatically create a connection to the
-	relevant database; in the example above, a connection will be made to the
-	Variation database for human. The three parameters passed specify the
-	species, database and object type you require. Below is a non exhaustive
-	list of Ensembl Variation adaptors that are most often used
+Below is a non exhaustive list of Ensembl Variation adaptors that are most often used:
```
Comment (sarahhunt) [inline docs/htdocs/info/docs/api/variation/variation_tutorial.html]: Can we remove ReadCoverage from here. I am confident it isn't frequently used!

Comment (ens-emily) [reply docs/htdocs/info/docs/api/variation/variation_tutorial.html]: I was thinking it didn't look popular!

**Pull request merged**